modules/hcp-shared-infra.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+// * hosted-control-planes/hcp-prepare/hcp-sizing-guidance.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-shared-infra_{context}"]
+= Shared infrastructure between hosted and standalone control planes
+
+As a service provider, you can more effectively use your resources by sharing infrastructure between a standalone {product-title} control plane and {hcp}. A 3-node {product-title} cluster can be a management cluster for a hosted cluster. 
+
+Sharing infrastructure can be beneficial in constrained environments, such as in the following situations:
+
+* You have only a few large bare-metal machines, and you want to use them as efficiently as possible
+* You want to implement more secure boundaries between your applications and your management plane
+
+Before you share infrastructure, ensure that your infrastructure can support the load of {hcp}. On the {product-title} management cluster, nothing else can be deployed except {hcp}. Workload must not be demanding, and it must fall within a low queries-per-second (QPS) profile.
```

Comment:
we should add that its currently limited to up to 3 hosted clusters shared to maintain a reasonable risk profile.

Reply:
Done